packages/kbn-visualization-utils/src/types.ts

```diff
@@ -50,9 +50,10 @@ export enum ChartType {
   Line = 'Line',
   Area = 'Area',
   Donut = 'Donut',
-  Heatmap = 'Heat map',
+  Heatmap = 'Heatmap',
```

Comment:
Is it safe to make these changes? How did it work before then?

Reply:
yes it is, it checks the title which is "Heat map" or the id which is "Heatmap" 👍

Reply:
Yes it looks for the title and the visualization id 👍

Reply:
Is safe because this enum is only used in within this context of changes (lens suggestions) and its usage is very limited now